src/video/rpi_cam.cc

```diff
@@ -0,0 +1,115 @@
+// BoB robotics includes
+#include "common/macros.h"
+#include "video/rpi_cam.h"
+
+// Standard C includes
+#include <cstring>
+
+// Standard C++ includes
+#include <stdexcept>
+
+// Extra POSIX includes
+#ifdef unix
```

Comment:
I'm not sure this is correct. Personally I'd do:
```c++
#ifdef _WIN32
#include <Winsock2.h>
#else
#include <fncntl.h>
#endif
```

Reply:
We are including winsock2.h elsewhere. I *think* the ``unix`` macro is standard.